pkg/runner/expression.go

```diff
@@ -512,7 +512,7 @@ func getEvaluatorInputs(ctx context.Context, rc *RunContext, step step, ghc *mod
 			for k, v := range config.Inputs {
 				value := nestedMapLookup(ghc.Event, "inputs", k)
 				if value == nil {
-					v.Default.Decode(&value)
+					_ = v.Default.Decode(&value)
```

Comment:
Any special reason to use _? Instead of keeping `v.Default.Decode(&value)` we are not using `_` anyways? Just curious...

Reply:
Ah gotcha! yeah with that convention the `_` has a lot of sense semantically. Thanks a lot for explaining this to my.